florist/api/servers/utils.py

```diff
@@ -0,0 +1,185 @@
+"""Utilities functions and definitions for starting a server."""
+from functools import partial
+from typing import Callable, Dict, List, Tuple, Union
+
+from fl4health.client_managers.base_sampling_manager import SimpleClientManager
+from fl4health.reporting.metrics import MetricsReporter
+from fl4health.server.base_server import FlServer
+from flwr.common.parameter import ndarrays_to_parameters
+from flwr.common.typing import Metrics, Parameters
+from flwr.server.strategy import FedAvg
+from torch import nn
+
+
+FitConfigFn = Callable[[int], Dict[str, Union[bool, bytes, float, int, str]]]
+
+
+def metric_aggregation(all_client_metrics: List[Tuple[int, Metrics]]) -> Tuple[int, Metrics]:
```

Comment:
I recently moved these functions (metric_aggregation and some of the others below) into fl4health (from the examples folder in fl4health) so they should be accessible by florist if you want to use them in place of redefining here. It will require updating to the most recent FL4Health version on pypi if you chose to do so

Reply:
Oh that's great, will update